pom.xml

```diff
@@ -221,12 +229,16 @@
                                 <repositoryUrl>file://${project.build.directory}/artifacts</repositoryUrl>
                             </repositoryUrls>
                             <launcherArguments>
+                                <vm-options>
```

Comment:
Please make this part of a profile if needed otherwise we may get into port clashes, especially on Jenkins.

Reply:
I commented the vm-options. I use them when I start sling with `mvn feature-launcher:start feature-launcher:stop -Dfeature-launcher.waitForInput=true`. It looks to me that a new profile cannot help in this situation when we do not need to make a real build. When added a new `debug` profile and started Sling with: `mvn feature-launcher:start feature-launcher:stop -Dfeature-launcher.waitForInput=true -Pdebug` the additional parameters were not taken.